changelog/changelog17425.dd

```diff
@@ -0,0 +1,18 @@
+add __traits(getParameterStorageClasses, f, i)
+
+$(LINK2 https://issues.dlang.org/show_bug.cgi?id=17425, Bugzilla 17425)
```

Comment:
The changelog entry seems a bit terse. I'd use at least more descriptive variable names. Also, linking to the documentation instead of bugzilla might be more helpful.